tests/utilities/test_remote_filesystem.py

```diff
@@ -0,0 +1,106 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+import fsspec
+import pytest
+
+from pytorch_lightning import Trainer
+from pytorch_lightning.callbacks import ModelCheckpoint
+from pytorch_lightning.loggers import TensorBoardLogger
+from tests.helpers import BoringModel
+
+GCS_BUCKET_PATH = os.getenv("GCS_BUCKET_PATH", None)
```

Comment:
@kaushikb11 These tests don't ever run, no? Because this environment variable does not get set anywhere in our CI.

Was it intended that this is set eventually? I would suggest dropping these tests as there's no guarantee that they work.